cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/distributed/DistributedProgramRuntimeService.java

```diff
@@ -349,7 +349,7 @@ public ProgramLiveInfo getLiveInfo(Id.Program program, ProgramType type) {
     public ClusterResourceReporter(MetricsCollectionService metricsCollectionService, Configuration hConf,
                                    CConfiguration cConf) {
       super(metricsCollectionService.getCollector(
-        ImmutableMap.of(Constants.Metrics.Tag.NAMESPACE, Constants.SYSTEM_NAMESPACE)));
```

Comment:
are there other stats being reported here?  the sendMetrics is called only once in this class and i am adding the default namespace before that in it's context.